experimental/dds/tree2/src/test/shared-tree/undo.spec.ts

```diff
@@ -133,178 +130,199 @@ const testCases: {
 		},
 		initialState: [{ child: "x" }],
 		editedState: [],
-		// Undoing the insertion of A on the parent branch is a no-op because the node was deleted
-		parentUndoState: [],
-		forkUndoState: [{}],
+		undoState: [{}],
+		// TODO: when acting on the view, the undo state is [{ child: "x" }]
```

Comment:
@yann-achard-MS This seems like a bug with editing data that's been concurrently deleted. We expect that the delete of "x" is done on the repair data after it's been moved to a detached field, right?

Reply:
Not yet, we haven't changed the merge semantics to make that happen yet.

Reply:
Ah ok, cool. What's the item for that? I'll add it to the todo.

Reply:
[Task 5111 Effective insert and removals under removed nodes](https://dev.azure.com/fluidframework/internal/_workitems/edit/5111)